include/irr/asset/CTexturePacker.h

```diff
@@ -94,13 +94,13 @@ class ITexturePacker
     ITexturePacker(uint32_t _pgSzxy = 256u, uint32_t _tilesPerDim = 32u, uint32_t _numLayers = 4u) :
         m_pgSzxy(_pgSzxy),
         m_tilesPerDim(_tilesPerDim),
-        m_addrAlctr(ADDR_LAYER_SHIFT>>1, _tilesPerDim, _numLayers)
+        m_pgtAddrAlctr(ADDR_LAYER_SHIFT>>1, _tilesPerDim, _numLayers)
     {
         assert(core::isPoT(_tilesPerDim));
         assert(core::isPoT(_pgSzxy));//because of packing mip-tail
     }
 
-    core::smart_refctd_dynamic_array<STexOffset> alloc(const ICPUImage* _img, const ICPUImage::SSubresourceRange& _subres)
+    STexOffset alloc(const ICPUImage* _img, const ICPUImage::SSubresourceRange& _subres)
     {
         auto extent = _img->getCreationParameters().extent;
         uint32_t levelCount = 1u;
```

Comment:
you don't need this variable anymore it seems?

Reply:
yea, thx